coingecko/constants.ts

```diff
@@ -1,10 +0,0 @@
-export const millisecondsInDay = 86400000;
-export const millisecondsInMinute = 60000;
-
-export const productStartInSeconds = 1653350400;
-export const productStartInMilliseconds = 1653350400 * 1000; //1653356522
-
-export const refreshInterval = 5; //minutes
-export const precision = 8;
-
-export const userKey = "user";
```

Comment:
this is not a userKey, this is a cacheKey

+ our constants could use a little uppercase love